packages/parsley/parsley.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+url {
```

Comment:
I see in your `opam` file (https://github.com/ghilesZ/parsley/blob/75ff33aeccc5c0fc22e8e1b55b1711f4d18c4d28/parsley.opam) that you defined the `url` field directly in the source repository. This is probably not what you want. `opam-publish` is here to do that for you and doing it the way you do might cause problem for people willing to try the dev version of your package.

Could you remove this field from your source directory (+ upcoming comments) and calling `opam publish` again?

Reply:
The `packages` directory in your repository is also unneeded (https://github.com/ghilesZ/parsley/tree/master/packages/parsley/parsley.0.0.1)

Reply:
Done. Thanx for your time and remarks